backend/src/sync.js

```diff
@@ -91,6 +91,43 @@ async function saveBlocks(blocksInfo) {
                     });
                   })
                 ),
+                models.AccessKey.bulkCreate(
+                  blockInfo.transactions.flatMap(tx => {
+                    const accountId = tx.receiver_id;
+                    return tx.actions
+                      .filter(action => action.AddKey !== undefined)
+                      .map(action => {
+                        let accessKeyType;
+                        if (
+                          typeof action.AddKey.access_key.permission ===
+                          "string"
+                        ) {
+                          accessKeyType = "FullAccess";
+                        } else if (
+                          action.AddKey.access_key.permission !== undefined
+                        ) {
+                          accessKeyType = Object.keys(
+                            action.a.access_key.permission
```

Comment:
1. It is a typo here in `.a.`
2. Let's extract the `permission` into a variable before the `if`s to simplify the code (it will most certainly collapse multiline wraps)